clients/vscode/package.json

```diff
@@ -94,6 +103,7 @@
     "axios": "^1.3.4",
     "events": "^3.3.0",
     "form-data": "^4.0.0",
-    "process": "^0.11.10"
+    "process": "^0.11.10",
+    "yarn": "^1.22.19"
```

Comment:
Prefer to install yarn globally rather than adding it as a dependency in package.json.

Reply:
Local yarn dependency removed.